src/core.js

```diff
@@ -1819,20 +1894,13 @@ define([
             mugs = multiselect ? mug : [mug],
             $baseToolbar = $(question_toolbar({
                 comment: multiselect ? '' : mug.p.comment,
-                isDeleteable: mugs && mugs.length && _.every(mugs, function (mug) {
-                    return _this.isMugRemoveable(mug, mug.hashtagPath);
-                }),
                 isCopyable: !multiselect && mug.options.isCopyable,
             }));
-        $baseToolbar.find('.fd-button-remove').click(function () {
-            var mugs = _this.getCurrentlySelectedMug(true, true);
-            form.removeMugsFromForm(mugs);
-            _this.refreshCurrentMug();
-        });
```

Comment:
It does...which isn't good. I'll put the delete button back for the multiple-questions-selected window at least (where the copy-paste help is shown).

I'm debating whether or not to put it back in question properties, too. I've been wondering if it would be good to get rid of the top bar in question properties altogether. Suspect it depends on how often people use the question type changer.